libr/core/cconfig.c

```diff
@@ -3151,6 +3151,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETCB ("cfg.sandbox", "false", &cb_cfgsanbox, "Sandbox mode disables systems and open on upper directories");
 	SETPREF ("cfg.wseek", "false", "Seek after write");
 	SETCB ("cfg.bigendian", "false", &cb_bigendian, "Use little (false) or big (true) endianness");
+	SETI ("cfg.cpuid", 0, "Run on cpuid");
```

Comment:
Shouldnt this be -1? We aim to go multithread at some point. Whats the point of being stick to one cpu? How can this be unset?

Reply:
that s a fair point.